spec/lib/sidebar_section_links_updater_spec.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+RSpec.describe SidebarSectionLinksUpdater do
+  fab!(:user) { Fabricate(:user) }
+  fab!(:user2) { Fabricate(:user) }
+
+  describe '.update_category_section_links' do
+    fab!(:public_category) { Fabricate(:category) }
+    fab!(:public_category2) { Fabricate(:category) }
+
+    fab!(:secured_category) do |category|
```

Comment:
Interesting, I usually just `Fabricate(:private_category, group: group)` for this case, is there much difference?

Reply:
O interested, I didn't know about the fabricator. I've updated it to your suggestion.